torchdynamo/utils.py

```diff
@@ -784,7 +784,6 @@ def same(
                     log.error(
                         f"RMSE (res-fp64): {res_error:.5f}, (ref-fp64): {ref_error:.5f}"
                     )
-                    # import pdb; pdb.set_trace()
```

Comment:
Was this left behind as a hint to folks to debug accuracy errors? this might be worth keeping...

Reply:
Will keep.